content/hardware/02.hero/boards/uno-wifi-rev2/tutorials/uno-wifi-rev-2/uno-wifi-r2-chromebook-installation/uno-wifi-r2-chromebook-installation.md

```diff
@@ -93,26 +99,26 @@ The above commands utilizes a tool called **AVRDUDE**, which is included in each
 **Windows:**
 
 ```
-"C:/Program Files (x86)/Arduino/hardware/tools/avr/bin/avrdude.exe"
+"%userprofile%/Desktop/avrdude/bin/avrdude.exe"
 ```
 
 **Mac:**
 
 ```
-/Applications/Arduino.app/Contents/Java/hardware/tools/avr/bin/avrdude
+/Users/$(whoami)/Desktop/avrdude/bin/avrdude
 ```
 
 **Linux:**
 
 ```
-/home/$(whoami)/Downloads/arduino-1.8.19-linux64/arduino-1.8.19/hardware/tools/avr/bin/avrdude
+/Users/$(whoami)/Desktop/avrdude/bin/avrdude
 ```
 
```

Comment:
Maybe its good to add the link to the HC article that talks about how to work on Chromebooks 
https://support.arduino.cc/hc/en-us/articles/360021251499-How-to-use-the-Web-Editor-with-Chromebook